opendcs-rest-api-jetty/build.gradle

```diff
@@ -46,15 +44,26 @@ task startJetty(type: JavaExec) { ->
             "-s")
 }
 
-task prepareStartScript(type: Copy) {
-    from './bin/start.sh'
-    into "${buildDir}/odcsapi-jetty-standalone/bin/"
-    filter(ReplaceTokens, tokens: ['jarName': jar.archiveFileName.get()])
+task createStartScript() {
+    doLast {
+
+        //Make bin directory if it does not exist already.  buildDir is now deprecated.
+        File buildDir = project.layout.getBuildDirectory().get().getAsFile();
```

Comment:
There is probably a better way to do this.  project.buildDir was deprecated, so this was one way to get it done.  Looking into a less convoluted way.

Reply:
Honestly it's probably fine. there are ways to properly define inputs and outputs from tasks but in this context that's probably going to look even more convoluted than what you have..

Reply:
Hmm, actually maybe not. This is similar is concept: https://github.com/USACE/cwms-data-api/blob/95735d01d6ddada96b8eb8aa53a07b392197548f/cwms-data-api/build.gradle#L232

You might consider this style.

Reply:
Looks like this one is using "$buildDir" (A much simpler approach) - which was deprecated in 8.5.
https://github.com/gradle/gradle/issues/20210

Reply:
we really need to update that particular build. still gradle 6.4